cli/src/main/java/org/jboss/as/cli/handlers/module/ASModuleHandler.java

```diff
@@ -707,4 +716,38 @@ public static void copy(final File source, final File target) throws CommandLine
         }
     }
 
-}
+    private String getSchemaVersion(CommandContext ctx, boolean hasSlot) {
+        if (detectedSchemaVersion == null) {
+            MXParser parser = new MXParser();
+            final File wfCommonModule = new File(modulesDir, "system.layers.base.org.wildfly.common.main.".replace('.', File.separatorChar) + "module.xml");
+            try {
+                parser.setFeature(FEATURE_PROCESS_NAMESPACES, true);
+                parser.setInput(new FileInputStream(wfCommonModule), null);
+                parser.nextTag();
+            } catch (Exception ignore) {
+            }
+            detectedSchemaVersion = parser.getNamespace();
```

Comment:
If an error occured, seems strange to access the parser. This should be guarded by the exception handling.